tests/views/playingScreen-test.js

```diff
@@ -37,16 +38,62 @@ describe('PlayingScreen', function () {
     TestUtils.Simulate.mouseUp(screen[0]);
     expect(clicked).toBe(true);
   });
+  it('creates a PlayingScreen and checks mouseDown, mouseUp with video360', function() {
+    var isTouched = false
+      , isStartHideControlBarTimer = false;
+    var mockController = {
+      state: {
+        isMobile: false,
+        accessibilityControlsEnabled: false,
+        upNextInfo: {
+          showing: false
+        },
+        isVideo360: true,
+        viewingDirection: {yaw: 0, roll: 0, pitch: 0}
+      },
+      startHideControlBarTimer: function () {
+        isStartHideControlBarTimer = true;
+      },
+      onTouched: function() { isTouched = true; },
```

Comment:
althought it is small, please change it to 
onTouched: function() { 
  isTouched = true; 
},


It's much easier to step through this using the debugger this way.